src/llama_stack_provider_kft/kft_adapter.py

```diff
@@ -296,8 +297,7 @@ async def get_training_job_status(self, job_uuid: str) -> Optional[PostTrainingJ
              scheduled_at=job.scheduled_at,
              started_at=job.started_at,
              completed_at=job.completed_at,
-             checkpoints=self._get_checkpoints(job),
-             resources_allocated=self._get_resources_allocated(job),
```

Comment:
is this removal expected?

Reply:
yep, we don't have a `_get_checkpoints` nor `_get_resources_allocated` so for now, we cannot set them in the status

Reply:
nvm missed your message in the description